src/alpaca_eval/models_configs/baichuan-13b-chat/configs.yaml

```diff
@@ -0,0 +1,13 @@
+baichuan-13b-chat:
+  prompt_template: "baichuan-13b-chat/prompt.txt"
+  fn_completions: "huggingface_local_completions"
+  completions_kwargs:
+    model_name: "./Baichuan-13B-Chat"
```

Comment:
can this directly be loaded from HF via `baichuan-inc/Baichuan-13B-Chat` model name? if so, could we update this?

Reply:
> can this directly be loaded from HF via `baichuan-inc/Baichuan-13B-Chat` model name? if so, could we update this?

Yes, the model can be loaded from HF via baichuan-inc/Baichuan-13B-Chat. I have updated the "./Baichuan-13B-Chat" to "baichuan-inc/Baichuan-13B-Chat"